src/components/Services.jsx

```diff
@@ -66,4 +69,4 @@ const mapStateToProps = state => {
   }
 }
 
-export default connect(mapStateToProps)(translate()(Services))
+export default connect(mapStateToProps)(translate()(withClient(Services)))
```

Comment:
Makes me think that we should probably have a `useClient` hook that we can use directly in useAppsInMaintenance hook.

Reply:
I was thinking the same while adding `withClient` in this PR, not enough time to investigate this one though. Maybe it's as easy as using `useContext` somehow?

Reply:
Seems like we would have to use the new Context via createContext to be able to use a hook. Issue created : https://github.com/cozy/cozy-client/issues/544